### ISSUE TITLE
Code alignment desired

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#124]
The following code should be aligned a little bit:

```
if bJsonSchema:
    try:
        validate(instance=oJsonCfgData, schema=oJsonSchemaCfg)
    except Exception as error:
        if error.validator == 'additionalProperties':
            logger.error(f"Verification against JSON schema failed: '{error.message}'\n" + \
                         "          Additional properties are not allowed! Please put the additional params into 'params': { 'global': {...} \n")
            BuiltIn().unknown('JSON schema validation failed!')
        elif error.validator == 'required':
            logger.error(f"The parameter {error.message}, but it's not set in JSON configuration file.\n")
            BuiltIn().unknown('JSON schema validation failed!')
        else:
            errParam = error.path.pop()
            logger.error(f"Parameter '{errParam}' with invalid value found in JSON configuration file! \n" + \
                         f"          {error.message}\n")
            BuiltIn().unknown('JSON schema validation failed!')
```

This is redundant: `BuiltIn().unknown('JSON schema validation failed!')`. You should put this as common final feedback at end of `except`, but outside all `if/elif`.

The error message in first case is:

```
[ ERROR ] Verification against JSON schema failed: 'Additional properties are not allowed ('Minimum_versionXYZ' was unexpected)'
          Additional properties are not allowed! Please put the additional params into 'params': { 'global': {...}
```
Seems that "`Additional properties are not allowed`" is already part of '`error.message`' - and therefore should be removed from second line in `logger.error()`.

**Comment by namsonx** [comment]
Hello Holger,

I created the pull-request #129 for this ticket.

Thank you,
Son

**Comment by test-fullautomation** [comment]
solved and already merged with #129
